cloudinit/cmd/status.py

```diff
@@ -61,23 +66,22 @@ def handle_status_args(name, args):
     # Read configured paths
     init = Init(ds_deps=[])
     init.read_cfg()
-
-    status, status_detail, time = _get_status_details(init.paths)
+    status, status_detail, time = get_status_details(init.paths)
     if args.wait:
-        while status in (STATUS_ENABLED_NOT_RUN, STATUS_RUNNING):
+        while status in (UXAppStatus.ENABLED_NOT_RUN, UXAppStatus.RUNNING):
             sys.stdout.write(".")
             sys.stdout.flush()
-            status, status_detail, time = _get_status_details(init.paths)
+            status, status_detail, time = get_status_details(init.paths)
             sleep(0.25)
         sys.stdout.write("\n")
     if args.long:
-        print("status: {0}".format(status))
+        print("status: {0}".format(status.value))
         if time:
             print("time: {0}".format(time))
         print("detail:\n{0}".format(status_detail))
     else:
-        print("status: {0}".format(status))
-    return 1 if status == STATUS_ERROR else 0
+        print("status: {0}".format(status.value))
+    return 1 if status == UXAppStatus.ERROR else 0
```

Comment:
Nit: It seems odd to have a conditional in which both if and else start by doing the same thing. Perhaps something like this?
```
    print("status: {0}".format(status.value))
    if args.long:
        if time:
            print("time: {0}".format(time))
        print("detail:\n{0}".format(status_detail))
    return 1 if status == UXAppStatus.ERROR else 0
```